tests/Responses/Expected/exchange_rate/update_ratio_with_string_validation_fail_response.json

```diff
@@ -0,0 +1,15 @@
+{
+    "code": 400,
+    "message": "Validation Failed",
+    "errors": {
+        "children": {
+            "ratio": {
+                "errors": [
+                    "The ratio must be a number."
+                ]
+            },
+            "sourceCurrency": {},
```

Comment:
We should be consistent with empty keys convention. Here you have `"sourceCurrency": { }` but [here](https://github.com/Sylius/Sylius/pull/7543/files#diff-52dbde23be4165acdceb08dbe8559a74R11) line is broken 🐃